pkg/resources/operation.go

```diff
@@ -238,11 +238,6 @@ func upgrade(
 	}
 
 	if opts.Draft {
-		// Only stopped resource can save as draft.
-		if !status.ResourceStatusStopped.IsTrue(entity) {
```

Comment:
Why remove it?

Reply:
The input resource comes from req.Model() which status is empty, can not using this to validation.